lib/bolt/bolt_option_parser.rb

```diff
@@ -748,7 +748,7 @@ def initialize(options)
         @options[:tty] = tty
       end
       define('--tmpdir DIR', 'The directory to upload and execute temporary files on the target') do |tmpdir|
-        @options[:tmpdir] = tmpdir
+        @options[:tmpdir] = File.expand_path(tmpdir)
```

Comment:
This path is used on the target, so expanding it locally doesn't make sense. Practically, it needs to be an absolute path but there's no easy way for us to test that (since absolute depends on whether the target is windows or *nix).

Reply:
Well, that's what I get for testing on localhost :P

Reply:
"Works with the local transport" is the new "works on my machine"